app/steps/medicaid/step_navigation.rb

```diff
@@ -24,6 +24,7 @@ class StepNavigation
       "Health" => [
         Medicaid::HealthDisabilityController,
         Medicaid::HealthPregnancyController,
+        Medicaid::HealthRecentPregnancyController,
```

Comment:
I think in general we've been following a pattern that would have this controller named `HealthPregnancyMemberController`

Reply:
k